### PR TITLE
[feature] component to manage patient identifiers

### PR DESCRIPTION
**Pull request by Ewanjiru**
## JIRA TICKET NAME:

PDMO-5: [Component for managing patient Identifiers](https://issues.openmrs.org/browse/PDMO-5?filter=-3)

## SUMMARY:

This component will consume REST api to display the existing patient identifiers and the identifier types.
It will also include functionality for adding,editing,deleting and undeleting identifiers.

### CODE REVIEW

File: app/js/components/patient.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import React from 'react';
 import Collapsible from 'react-collapsible';
 import apiCall from '../utilities/apiHelper';
 import Addresses from './addresses'
+import Identifiers from './Identifiers';
```
Comment (dkayiwa) [inline app/js/components/patient.jsx]: In the above './Identifiers', did you change the convention to start with capital I?

File: coverage/testSetup.js.html [hunk 2]
```diff
@@ -1,191 +0,0 @@
...
-    getItem() { <span class="cstat-no" title="statement not covered" >return undefined; </span>},
-    setItem() {},
-    removeItem() { <span class="cstat-no" title="statement not covered" >return undefined; </span>},
-  };
-}
-global.window = window;
-global.$ = spy(<span class="fstat-no" title="function not covered" >()</span> =&gt; (<span class="cstat-no" title="statement not covered" >{</span>
-  validate: <span class="fstat-no" title="function not covered" >(c</span>ontext) =&gt; {
-<span class="cstat-no" title="statement not covered" >    Object.keys(context.rules).forEach(<span class="fstat-no" title="function not covered" >ke</span>y =&gt; <span class="cstat-no" title="statement not covered" >new Promise(<span class="fstat-no" title="function not covered" >(r</span>esolve) =&gt; {</span></span>
-<span class="cstat-no" title="statement not covered" >      if (context.rules[key].required) {</span>
```
Comment (dkayiwa) [inline coverage/testSetup.js.html]: Is this also related to this ticket?

File: coverage/sorter.js [hunk 3]
```diff
@@ -1,158 +0,0 @@
...
-            col,
-            i;
-
-        for (i = 0; i < colNodes.length; i += 1) {
-            colNode = colNodes[i];
-            col = {
-                key: colNode.getAttribute('data-col'),
-                sortable: !colNode.getAttribute('data-nosort'),
-                type: colNode.getAttribute('data-type') || 'string'
-            };
```
Comment (dkayiwa) [inline coverage/sorter.js]: Is this also related to this ticket?

**Comment by dkayiwa** [comment]
@Ewanjiru i see a couple of file changes or removals that do not seem related to this ticket. Can you look into it?

**Comment by Ewanjiru** [comment]
Looking into this.

**Comment by dkayiwa** [comment]
@Ewanjiru hopefully you have already seen the merge conflicts on this pull request.
